cpp/tests/large_strings/merge_tests.cpp

```diff
@@ -51,6 +51,9 @@ TEST_F(MergeTest, MergeLargeStrings)
     CUDF_TEST_EXPECT_COLUMNS_EQUIVALENT(c, input);
   }
 
+  // Unset the LIBCUDF_LARGE_STRINGS_THRESHOLD if already set.
```

Comment:
Setting env var to a smaller number may result in this test to fail (and rightfully so) so locally unsetting it here

Reply:
How does this happen? Is there some place outside this test where this is set but not unset?

Reply:
Not really. This is in case if one has set this env var to a smaller value while running other tests, they would start seeing this test fail inadvertently. Can revert this too!

Reply:
If another test is setting and not unsetting this I think I would consider that a bug and would want something to fail.
I was curious if you added this because it was failing.

Reply:
Oh no, I was just running another test by changing this env variable to do some experimentation and encountered this test failing too so I got a bit concerned.

Reply:
Removing this!